be/src/olap/tablet.cpp

```diff
@@ -1214,33 +1214,32 @@ void Tablet::do_tablet_meta_checkpoint() {
 
 bool Tablet::rowset_meta_is_useful(RowsetMetaSharedPtr rowset_meta) {
```

Comment:
Could you add a comment for `rowset_meta_is_useful`? what's the meaning of `useful`?

Why when `find_version` is false, the rowset is unuseful?

Reply:
@kangkaisen Comment added.

> Why when find_version is false, the rowset is unuseful?

I think you want to ask why when find_version is **true**, the rowset is unuseful. Because when version founded, means the tablet has the data in this version (maybe compacted), we can sweep up this rowset. Only when the rowset has some new data (version is not covered, i.e. find_version is false), the rowset is useful.

Reply:
OK, Thanks!